### PR TITLE
Add logs

### PR DESCRIPTION
**Pull request by kolya182**
Ignore generated logs.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,6 +11,11 @@ node_modules
...
 
 $ Ignore the .env file for the server
-.env.example
+.env
 
-dist
\ No newline at end of file
+# Ignore local logs
+access.log
+trustwallet.log
```
Comment (michaelScoff) [inline .gitignore]: could you do `*.log` instead?

File: .gitignore [hunk 2]
```diff
@@ -11,6 +11,11 @@ node_modules
...
+
+dist
+package-lock.json
```
Comment (michaelScoff) [inline .gitignore]: What's the deal with `package-lock.json` it sounds like they recommend to commit this one

File: .gitignore (hunk 1 above)
Comment (kolya182) [reply .gitignore]: Fixed, thanks.

File: .gitignore (hunk 2 above)
Comment (kolya182) [reply .gitignore]: Official docs controversial https://docs.npmjs.com/files/package-lock.json.  I thought to force it to be ignored.

**Pull request merged**